examples/camera/basic/buildozer.spec

```diff
@@ -31,8 +31,9 @@ source.include_exts = py,png,jpg,kv,atlas
 # (str) Application versioning (method 2)
 version = 1.0
 
-# (list) Application requirements
-requirements = plyer,kivy
+# (list) Application requirements 
+# android is also require for run this app on android
```

Comment:
Grammar mistake :P . `android library is also required to run this app on Android platform` might be better.